src/js/utils/confirmationModalUtils.jsx

```diff
@@ -28,6 +29,7 @@ const confirmationModal = ({
         hideCloseButton={hideCloseButton}
       />
     ),
+    closeOnClickOutside,
```

Comment:
just double checking: The behaviour before your change was to always close on click outside. Now it still defaults to that behaviour but you can specify for it to not by setting closeOnClickOutside = false.